### PR TITLE
Make the monster tokens hidden to the players by default.

### PR DESCRIPTION
**Pull request by Kokolokoli**
This PR is intended as a first step into the project. It changes and sets the default token settings inside the init_settings() function.

### CODE REVIEW

**Comment by Kokolokoli** [comment]
Didn't do a commit for it since it didn't require changes, but I did test in Google Chrome as well

**Review by vlaminck — COMMENTED** [review]
I don't think I understand what this accomplishes. It seems like it would prevent the "Reset Token Settings to Defaults" button from working properly. It seems like it would just make the button toggle the `hidden` feature instead of actually resetting all the settings to their default values.

**Comment by Kokolokoli** [comment]
Since the "Reset Token Settings to Defaults" button calls the click() of each toggle, and the click of each toggle sets the corresponding setting to the new value (See Settings.js line 125, which uses the function defined in SidebarPanel.js line 224).

Reset Token Settings essentially sets everything to false, except for the "hidden" setting where it sets it right back to true.

I removed the window.TOKEN_SETTINGS[name] = false for each toggle in the loop. This is what used to set the initial values when the init_settings was called, but then it needlessly set them a second time whenever clicking the button (since toggle.click already does that). 

Instead, I simply click the "Reset Token Settings to Defaults" after building it and appending it.

File: Settings.js [hunk 1]
```diff
@@ -156,12 +156,13 @@ function init_settings(){
...
 				toggle.click();
 			}
-			window.TOKEN_SETTINGS[setting.name] = false;
+			if (setting.name == 'hidden') toggle.click();
 		}
 		persist_token_settings(window.TOKEN_SETTINGS);
 		redraw_settings_panel_token_examples();
 	});
 	body.append(resetToDefaults);
+	resetToDefaults.click(); // Sets the default token settings
```
Comment (vlaminck) [inline Settings.js:165]: This line in particular just resets all the defaults on page load.

**Comment by vlaminck** [comment]
Other than that line that resets the defaults when the page loads, your logic is fine. I'm curious why we would want to start defaulting `hidden` to `true`, though. This is a departure from current expectations.

File: Settings.js (hunk 1 above)
Comment (Kokolokoli) [reply Settings.js:165]: Yes. Sorry I misunderstood that last part and corrected my above comment. However, the token settings still don't persist in the main branch (everything is false on load).

**Comment by Kokolokoli** [comment]
Extremely good remark.

I'm a big dummy, that's why. I completely misunderstood a perfectly written message. I was supposed to make "Restrict Player Movement" true by default, not "hidden".

https://discord.com/channels/815028457851191326/832199739398946817/931271695560507423

Will close this PR for now.

**Pull request closed without merging**